Libraries/src/Amazon.Lambda.AppSyncEvents/Amazon.Lambda.AppSyncEvents.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\..\buildtools\common.props" />
+
+  <PropertyGroup>
+    <Description>Amazon Lambda .NET Core support - AWS AppSync package.</Description>
+    <TargetFrameworks>netstandard2.0;netcoreapp3.1;net8.0</TargetFrameworks>
```

Comment:
Since this is a new package, we should only target `net8.0`

Reply:
NET 8 test was written in the 'EventsTests.NET6' project, so I included both net6.0 and net8.0

Reply:
We shouldn't add a "net6.0" target. For the tests, you could create a 'EventsTests.NET8' project and add the tests there.

Reply:
Added new project - EventsTests.NET8